ql/src/java/org/apache/hadoop/hive/ql/DriverTxnHandler.java

```diff
@@ -67,7 +67,6 @@
 import org.apache.hadoop.hive.ql.session.SessionState.LogHelper;
 import org.apache.hadoop.util.StringUtils;
 import org.apache.hive.common.util.ShutdownHookManager;
-import org.apache.hive.common.util.TxnIdUtils;
```

Comment:
This is a good change, but please revert for this PR to keep it focused on the Driver classes.

Reply:
OK, removed.